deploy/backplane/mtsre/10-mtsre-monitoring-coreos-apis.ClusterRole.yml

```diff
@@ -8,6 +8,7 @@ rules:
       - "monitoring.coreos.com"
     resources:
       - servicemonitors
+      - prometheuses
```

Comment:
Is write access required to these?

Reply:
whoops! good catch, let me make it read-only

Reply:
https://github.com/openshift/managed-cluster-config/pull/1246#issuecomment-1216236222

Here's the cause Christoph, and in such situations, we require visibility as well as ways to test any updates on the fly to see ultimately how the Prometheus CR should be configured so as to make it operate properly.